lib/security/ci-security-groups.ts

```diff
@@ -26,7 +26,13 @@ export class JenkinsSecurityGroups {
       vpc,
       description: 'Main node of Jenkins',
     });
+
+    this.agentNodeSG = new SecurityGroup(stack, 'AgentNodeSG', {
```

Comment:
move the declaration and the additional rules after the main agent nodes, so it reads more like 

* External Access SG / Rules 
* MainNode SG / Rules
* AgentNode SG / Rules

Reply:
Done

Reply:
Done